src/main/webapp/package.json

```diff
@@ -14,16 +14,16 @@
   "dependencies": {
     "@angular/animations": "^11.0.9",
     "@angular/cdk": "^11.1.0",
-    "@angular/common": "^11.0.9",
+    "@angular/common": "^11.1.2",
     "@angular/compiler": "^11.0.9",
-    "@angular/core": "^11.0.9",
+    "@angular/core": "^11.1.2",
     "@angular/flex-layout": "^11.0.0-beta.33",
     "@angular/forms": "^11.0.9",
     "@angular/google-maps": "^11.1.1",
     "@angular/material": "^11.1.0",
-    "@angular/platform-browser": "^11.0.9",
+    "@angular/platform-browser": "^11.1.2",
     "@angular/platform-browser-dynamic": "^11.0.9",
-    "@angular/router": "^11.0.9",
+    "@angular/router": "^11.1.2",
```

Comment:
Could you revert back to the previous versions of the dependencies? Unless you specifically need the latest versions for what you're doing, it might be better to keep the versions we had previously (since we know it works with them!)

Reply:
okay I will revert back to them no worries